cli/Valet/Site.php

```diff
@@ -212,13 +212,53 @@ public function secure($url)
     {
         $this->unsecure($url);
 
+        $this->files->ensureDirExists($this->caPath(), user());
+
         $this->files->ensureDirExists($this->certificatesPath(), user());
 
+        $this->createCa();
+
         $this->createCertificate($url);
 
         $this->createSecureNginxServer($url);
     }
 
+    /**
+     * If CA and root certificates are nonexistent, crete them and trust the root cert.
+     *
+     * @return void
+     */
+    public function createCa()
+    {
+        $caPemPath = $this->caPath() . '/LaravelValetCASelfSigned.pem';
+        $caKeyPath = $this->caPath() . '/LaravelValetCASelfSigned.key';
+
+        if ($this->files->exists($caKeyPath) && $this->files->exists($caPemPath)) {
+            return;
+        }
+
+        $oName = 'Laravel Valet CA Self Signed Organization';
+        $cName = 'Laravel Valet CA Self Signed CN';
+
+        if ($this->files->exists($caKeyPath)) {
+            $this->files->unlink($caKeyPath);
+        }
+        if ($this->files->exists($caPemPath)) {
+            $this->files->unlink($caPemPath);
+        }
+
+   //     $this->cli->run(sprintf(
```

Comment:
Don't include commented out code unless absolutely necessary for some specific reference and please note why it is being left in the code.

Reply:
Does any of the certutil stuff need to remain? We are only concerned with installing certs Windows side?

Reply:
Not in the linux side no. Unless someone is running a browser from linux using some kind of x util that sends the gui to windows for rendering you don't have to worry about the store on linux.

Reply:
In which case, I believe that [this method](https://github.com/valeryan/valet-wsl/blob/master/cli/Valet/Site.php#L290) and [its call](https://github.com/valeryan/valet-wsl/blob/master/cli/Valet/Site.php#L244) could also be removed.